src/utils/newRelease.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import newReleaseUtil from './newRelease';
+import FlashMessageHolder from '../components/toolbox/flashMessage/holder';
+
+describe('new release util', () => {
+  const callbacks = {};
+  const ipc = {
+    on: jest.fn((event, callback) => { callbacks[event] = callback; }),
+    send: jest.fn(),
+  };
+
+  beforeEach(() => {
+    ipc.send.mockClear();
+    window.ipc = ipc;
+  });
+
+  it('Should return undefined if no ipc on window', () => {
+    delete window.ipc;
```

Comment:
`delete window.ipc;` should happen `afterEach` so that this test doesn't affect other (future) tests.